Registry/awscoreconfiguration.setreg

```diff
@@ -0,0 +1,10 @@
+{
+    "Amazon": 
+    {
+        "AWSCore": 
+        {
+            "ProfileName": "default",
+            "ResourceMappingConfigFileName": "default_aws_resource_mappings.json"
+        }
+    } 
+}
```

Comment:
Nitpick missing newline at the end of this fine.

Reply:
Fixed!